worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -596,6 +625,38 @@ private[celeborn] class Worker(
       })
     }
 
+  private def removeAppResourceConsumption(applicationId: String): Unit = {
+    userResourceConsumptions.asScala.foreach {
```

Comment:
userResourceConsumptions may not contain the application info ,then The gauge of the application in ResourceConsumptionSource can't be removed